### PR TITLE
sys: fmt: fix scn_u32_dec() documentation

### PR DESCRIPTION
**Pull request by kaspar030**

### CODE REVIEW

File: sys/include/fmt.h [hunk 1]
```diff
@@ -214,10 +214,10 @@ size_t fmt_str(char *out, const char *str);
  *
  * Will convert up to @p n digits. Stops at any non-digit or '\0' character.
  *
- * @param[out]  str  Pointer to string to read from
+ * @param[in]   str  Pointer to string to read from
  * @param[in]   n    Maximum nr of characters to consider
```
Comment (miri64) [inline sys/include/fmt.h]: I think "nr" is a german abbreviation (the English equivalent is "no."). While you at it: can you fix it?

Comment (miri64) [inline sys/include/fmt.h]: (or if you go for American English you can also go "#", but I think that's confusing ;-))

**Comment by miri64** [comment]
> if the string does not contain any digits e.g.. But that's unrelated to this PR.

Without knowing the code, the conventional thing would be to just return 0 and not an error, so I guess it does that.

File: sys/include/fmt.h (hunk 1 above)
Comment (kaspar030) [inline sys/include/fmt.h]: As that's in all the doc strings, it'll have to wait.

**Pull request merged**